config/software/datadog-agent.rb

```diff
@@ -10,6 +10,12 @@
   source path: ENV['LOCAL_AGENT_REPO']
 end
 
+if ohai['platform'] == "windows"
+  integrations_dir="../integrations-core"
```

Comment:
Nit: can you add a space before and after the `=` sign? @olivielpeau should we run rubocop (with the minimum of rules)